tools/genkconfig.py

```diff
@@ -97,10 +97,12 @@ def main():
 
     section_mappings = {
         "servers": None,
+        "servers/drivers": None,
```

Comment:
I think you don't need to add "servers/drivers" entry here.

Reply:
I did this to avoid the change you requested below. But now as you said, I'll move the files to `servers/drivers/rtc/rtc`.